apps/dc/dev/base/kustomization.yaml

```diff
@@ -7,3 +7,4 @@ patches:
   #overlays for dc-purview-shir
   - path: ../../dc-purview-shir/dev.yaml
   - path: ../../serviceaccount/dev.yaml
+  - path: ../../identity/dc-purview-shir-identity-dev.yaml
```

Comment:
we usually just have the reference as <env>.yaml - this example `dev.yaml`

Reply:
Updated